src/core/ipfs/files.js

```diff
@@ -81,6 +82,10 @@ module.exports = function files (self) {
           return file
```

Comment:
We need to `pause` the stream? Would it otherwise start emitting without listeners?